src/app.ts

```diff
@@ -65,6 +66,14 @@ const defaultModule = {
   startIdpMetadataUpdater
 };
 
+const cache = apicache.options({
+  debug: false,
+  defaultDuration: "5 minutes",
```

Comment:
can we make this configurable via env var? (5 minutes default is fine)

Reply:
@cloudify like a required ENV var?

Reply:
no an optional env var, that defaults to 5 minutes if not set

Reply:
fixed in 4e2f145 and ddeacff